templates/base_index.html

```diff
@@ -78,6 +78,7 @@
 
     <script src="{{ versioned_static('js/navigation.js') }}" defer></script>
     <script src="https://assets.ubuntu.com/v1/703e23c9-lazysizes+noscript+native-loading.5.1.2.min.js" defer></script>
+    <script src="https://cdn.jsdelivr.net/npm/fuse.js@6.6.2"></script>
```

Comment:
Fuse is installed as a dependency in package.json, but then we link to a CDN hosted version of the script - could we add fuse to the `build-js` step, and link to that script instead?

Reply:
Also, does this need to be in the base template? Is there a careers specific template this could be moved to? Careers pages are the only ones that need to load this script.